fe/src/main/java/org/apache/doris/alter/Alter.java

```diff
@@ -276,8 +276,9 @@ public void processAlterTable(AlterTableStmt stmt) throws UserException {
             if (olapTable.getState() != OlapTableState.NORMAL) {
                 throw new DdlException("Table[" + table.getName() + "]'s state is not NORMAL. Do not allow doing ALTER ops");
             }
-            
-            if (needTableStable) {
+
+            // schema change job will wait until table become stable
+            if (needTableStable && !hasSchemaChange) {
```

Comment:
I think rollup job can be changed to same as schema change job.

Reply:
@morningman Done